### PR TITLE
User Feedback Team Meeting

### PR DESCRIPTION
**Pull request by mihaiep**
Minutes/notes

### CODE REVIEW

File: meetings/2018-09-14.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Links
+
+* **Recording**:https://www.youtube.com/watch?v=00YONf8MrIs
+* **GitHub Issue**: [91](https://github.com/nodejs/user-feedback/issues/91)
+* **Minutes Google Doc**: [Minutes](https://docs.google.com/document/d/1082uJlCb9ar9VThHI1ZRqfy9K2lpHtszSPGWkTfh53A/)
+
+## Present
+
+
+* User Feedback Members: @nodejs/user-feedback: Dan Shaw, Michael Dawson, Mihai Ene-Pietrosanu, Ahmad Nassri
```
Comment (mhdawson) [inline meetings/2018-09-14.md]: I think we could just make a bullet list.  I think Agiri is a member and Joe was looking to join as well.

File: meetings/2018-09-14.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* September 2018 Node.js Online Meetup [#90](https://github.com/nodejs/user-feedback/issues/90)
+* Evolving General User Feedback by hosting an online Node.js Meetup [#82](https://github.com/nodejs/user-feedback/issues/82)
+* Public User Feedback request - Promises [#77](https://github.com/nodejs/user-feedback/issues/77)
+* Survey for strong(er) typing [#61](https://github.com/nodejs/user-feedback/issues/61)
+
+## Notes
+* **Highlights:**
+  * We got a max of 14 people in the Meetup and about 140 people on the group! **Thank you Dan! Thank you Michael&Rich for the great presentations at the Meetup.** 
+  * We have a new potential memeber Joe Sepi. Thank you Joe! helping on issue [77](https://github.com/nodejs/user-feedback/issues/77)
+  * **Action item**: Michael follow up with Chris on Tooling (expectansions).
```
Comment (mhdawson) [inline meetings/2018-09-14.md]: expectansions -> expectations

**Comment by mihaiep** [comment]
Fixed, thank you Michael.

File: meetings/2018-09-14.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+* **Minutes Google Doc**: [Minutes](https://docs.google.com/document/d/1082uJlCb9ar9VThHI1ZRqfy9K2lpHtszSPGWkTfh53A/)
+
+## Present
+
+
+* User Feedback Members: @nodejs/user-feedback: 
+* Dan Shaw (@dshaw) 
+* Michael Dawson (@mhdawson)
+* Mihai Ene-Pietrosanu (@mihaiep)
+* Ahmad Nassri (ahmadnassri)
```
Comment (styfle) [inline meetings/2018-09-14.md]: Missing the `@` symbol for ahmadnassri

**Comment by mihaiep** [comment]
added  @ahmadnassri.

**Pull request merged**